services/masterdata/src/lib/components/masterdatas-edit-form/masterdatas-edit-form.component.html

```diff
@@ -150,6 +150,27 @@
 								</div>
 							</div>
 						</div>
+            <!-- tags -->
+            <div class="field">
+              <label for="tags">Tags</label>
+              <lens-services-tags-selector
+                id="tags"
+                formControlName="tags"
+                [tags]="
+                  tagsList
+                "></lens-services-tags-selector>
```

Comment:
Horrible formatting :). Currently we disabled format on save to prevent this ugly auto formatting so you could format manually

Reply:
fixed formatting! i will dedicate a task on formatting if i missed something on a future story!